.github/workflows/release-macos.yml

```
@@ -41,12 +41,19 @@ jobs:
                 rustup update --no-self-update stable
                 rustup default stable
 
+            - name: Setup for x86 cross-compiling
+              if: matrix.arch == 'x64'
+              run: |
+                # Install x86 homebrew
+                arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+                # Install x86 Rust target
+                rustup target add x86_64-apple-darwin
```

```suggestion
            - name: Install brew and Rust target
              run: |
                arch -${{matrix.arch_terminal}} /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

                rustup target add ${{ matrix.rust_target_prefix }}-apple-darwin
```

I think I would like the idea of making this fully general and splitting it into two steps of `Install brew` and `Install Rust target`, but they would look like what I have inserted as a suggestion here.

I realize it would probably be a no-op for the arm case, but we've already got those nice env vars set up...so it seems nice to use them in this way?

---

I think arm64 brew is already installed, and I wouldn't expect the install script to detect existing installs and decide to do a no-op? I could see it fail with an error. So I'd rather avoid calling it for arm64.

The rustup command is a no-op though.

I think the conditional `if:` step is clearer. And when we stop supporting intel, it can be removed without any other changes.